src/distilabel/utils/serialization_v2.py

```diff
@@ -29,7 +29,7 @@
 SaveFormats = Literal["json", "yaml"]
 
 
-def _get_class(module: str = None, name: str = None) -> Type:
+def _get_class(module: str, name: str) -> Type:
```

Comment:
The default args are placed so [here](https://github.com/argilla-io/distilabel/pull/366/files#diff-a566a0dfe56d6cf771f7e75367798fb6e01641133dce994665d5e256699a21a0R355) we can pass the data using **kwargs. But also, I see I have to update the code to use `distilabel/pipeline/serialization` and remove this file. Will do that in a different PR.

Reply:
oops, my bad!